docs/ruby/README.md

```diff
@@ -0,0 +1,15 @@
+# Ruby
+![](../images/ruby-logo.png)
+
+sqlcommenter support is available for the following [Ruby](https://www.ruby-lang.org/en/) ORMs/frameworks:
+
+[![](../images/activerecord_marginalia-logo.png)](ruby-on-rails)
+
+<style>
+    img[src*='/activerecord_marginalia-logo.png']{
+        max-width: 30%;
+        float: None;
+        margin:0 2%;
+    }
+
+</style>
```

Comment:
Github strips these tags and also there is no image `activerecord_marginalia-logo.png `

Reply:
@lonewolf3739 the image is there in the location: `docs/images/activerecord_marginalia-logo.png` in the commit section.

Also for reference: this is view when the doc is deployed with gh-pages: [sqlcommenter-doc-preview](https://thiyagu55.github.io/opentelemetry-sqlcommenter/ruby/)